src/serde.rs

```diff
@@ -406,6 +491,15 @@ mod tests {
             .boxed()
     }
 
+    fn arb_homogeneous_map(
+        key_gen: BoxedStrategy<MapKey>,
+        value_gen: BoxedStrategy<Value>,
+    ) -> BoxedStrategy<Value> {
+        prop::collection::hash_map(key_gen, value_gen, 1..10)
```

Comment:
when we support empty maps lets change this to `0..10`

Reply:
08d5194